### DISCUSSION TITLE
How do I send data to an S3 bucket after a dbt job?

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17527] [Q&A] [answered]
Newbie question here, I have 2 jobs, one that is based on dbt models and one that sends data to an s3 bucket, I want to orchestrate the jobs together such that the dbt models are ran and then subsequently the send to s3 happens. Can someone help me out with this?

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1698771936672019?thread_ts=1698771936.672019&cid=C01U954MEER).*

**Comment by tacastillo** [comment, marked as answer]
Hi!

With the dbt integration, you can make assets downstream of your dbt models.[ Here's an example of how it can be done.](https://docs.dagster.io/integrations/dbt/using-dbt-with-dagster/downstream-assets). In your case, you can write a downstream Python function that defines an asset for this export, define its dbt dependencies with `get_asset_key_for_model`, and then use a lib like boto or smart-open that queries the data and exports it to S3.

By doing this, you can combine your two jobs into one and have one job that selects your relevant dbt models and the exported S3 asset.
